src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,27 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, body, user, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+        <UserInfo user={user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {body}
+    </p>
+
+    <hr />
+
+    { comments && comments.length
+      ? <CommentList comments={comments} />
+      : <b data-cy="NoCommentsMessage">No comments yet</b>
+    }
```

Comment:
comments is array, so `Boolean([]) === true`
```suggestion
    {comments.length
      ? <CommentList comments={comments} />
      : <b data-cy="NoCommentsMessage">No comments yet</b>
    }
```